src/ImageSharp.Drawing/Pens/Pen{TPixel}.cs

```diff
@@ -26,16 +27,16 @@ namespace ImageSharp.Drawing.Pens
     public class Pen<TPixel> : IPen<TPixel>
         where TPixel : struct, IPixel<TPixel>
     {
-        private static readonly float[] EmptyPattern = new float[0];
-        private readonly float[] pattern;
+        private static readonly ReadOnlySpan<float> EmptyPattern = new float[0];
+        private readonly ReadOnlySpan<float> pattern;
```

Comment:
The situation raises an API question:

- You can take a `ReadOnlySpan<T>` and copy its data into a `Buffer<T>` inside.
- But in an other overload you can take a `float[]` without copying anything (faster).

I wonder if it's OK to implement a heterogenous API, or should we just ignore span for now on places like this.

FYI What you really need here is called [`ReadOnlyBuffer<T>`](https://github.com/dotnet/corefxlab/blob/master/src/System.Buffers.Primitives/System/Buffers/ReadOnlyBuffer.cs) in the future API-s. (Kind-of `ReadOnlySpan<T>` that be used as member.)